wsgi/odyssy/basic/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.conf.urls import url
 
-import views
+from . import views
```

Comment:
I don't know the reason, but it is showing some error on my machine. I tried to add current directory to PATH variable and some other methods suggested on internet, but it didn't worked. That's why I had to change. Anyone else facing same issue @Monal5031 @yashLadha ?